application/database/database_routes.py

```diff
@@ -245,11 +275,16 @@ def get_location(location_id):
   response.headers.add('Access-Control-Allow-Origin', '*')
   return response
 
-@database_bp.route('/api/v1/location/<id>', methods=['POST'])
+@database_bp.route('/api/v1/location/<id>', methods=['PUT'])
 @basic_auth.required
 def update_location(id):
+  """
+  Update a location
+  """
   response = make_response(jsonify("Not Implemented"), 501)
   response.headers.add('Access-Control-Allow-Origin', app.config['SITE_ENDPOINT'])
   response.headers.add('Access-Control-Allow-Headers', 'Authorization, Content-Type')
 
+  #todo: dont update created_on
+
```

Comment:
for implementation: the following fields are not allowed to be modified via the API: 
`created_on`
`record_id`
`location_id`

additionally, updates to the following fields should NOT be allowed to anyone outside our org, and (not via the FE)
`is_hidden`
`is_verified`
`deleted_on`
`external_location_id`